content/docs/start/experiments.md

```diff
@@ -4,237 +4,227 @@ title: 'Get Started: Experiments'
-provides a mechanism for stages to depend on the values of specific sections of
-such a config file (YAML, JSON, TOML, and Python formats are supported).
+```dvc
+$ dvc exp run --queue --set-param train.min_split=8
+Queued experiment 'd3f6d1e' for future execution.
+$ dvc exp run --queue --set-param train.min_split=64
+Queued experiment 'f1810e0' for future execution.
+$ dvc exp run --queue --set-param train.min_split=2 --set-param train.n_est=100
+Queued experiment '7323ea2' for future execution.
+$ dvc exp run --queue --set-param train.min_split=8 --set-param train.n_est=100
+Queued experiment 'c605382' for future execution.
+$ dvc exp run --queue --set-param train.min_split=64 --set-param train.n_est=100
+Queued experiment '0cdee86' for future execution.
+```
 
-Luckily, we should already have a stage with
-[parameters](/doc/command-reference/params) in `dvc.yaml`:
+Next, run all queued experiments simultaneously using `--run-all` (and in
+parallel with `-j`):
 
-```yaml
-featurize:
-  cmd: python src/featurization.py data/prepared data/features
-  deps:
-    - data/prepared
-    - src/featurization.py
-  params:
-    - featurize.max_features
-    - featurize.ngrams
-  outs:
-    - data/features
+```dvc
+$ dvc exp run --run-all -j 2
 ```
 
-<details>
+## Comparing many experiments
 
-### 💡 Expand to recall how it was generated.
+To compare all of these experiments, we need more than `diff`.
 
-The `featurize` stage
-[was created](/doc/start/data-pipelines#dependency-graphs-dags) with this
-`dvc run` command. Notice the argument sent to the `-p` option (short for
-`--params`):
+`dvc exp show` compares any number of experiments in one table:
 
 ```dvc
-$ dvc run -n featurize \
-          -p featurize.max_features,featurize.ngrams \
-          -d src/featurization.py -d data/prepared \
-          -o data/features \
-          python src/featurization.py data/prepared data/features
+$ dvc exp show --no-timestamp --include-params train.n_est,train.min_split
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2
+│ master        │  0.55259 │ 0.91536 │ 50         │ 2
+│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8
+│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64
+│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2
+│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8
+│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
 ```
 
-</details>
+Each experiment is given an arbitrary name by default (although we can specify
+one with `dvc exp run -n`.) We can see that `exp-98a96` performed best among
+both of our metrics, with 100 estimators and a minimum of 64 samples required to
+split a node.
```

Comment:
We have already shortened this param name and taken out any descriptions of the specific parameters, so I'm not sure it's going to make sense to anyone familiar with random forest models without the full description. Users may not really care what these hyperparameters mean, but if anyone does, they might be confused since there are multiple hyperparameters in a random forest model that take some minimum number of samples. See https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html.

Reply:
> Users may not really care what these hyperparameters mean, but if anyone does, they might be confused

K. Yeah this was 💅 but if you prefer to have it there let's keep it.